setup.py

```diff
@@ -177,7 +177,6 @@ def get_cmdclass():
         'Programming Language :: C',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3.2',
-        'Programming Language :: Python :: 3.3',
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
```

Comment:
Py3.7 is included in tox.ini test, here it's not.

Reply:
removed from tox.ini in 3bc78bc